lib/youtube-dl.js

```diff
@@ -305,3 +305,21 @@ ytdl.getSubs = function(url, options, callback) {
     callback(null, files);
   });
 };
+
+/**
+ * @params {Boolean} descriptions
+ * @param {Function(!Error, Object)} callback
+ */
+ytdl.getExtractors = function(descriptions, options, callback) {
```

Comment:
These are the child process execution options, it always makes sense to have them here IMHO - if only for the timeout option (http://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback)